src/ModelInit.jl

```diff
@@ -468,6 +472,15 @@ function get_relevant_sources{NumType <: Number}(
   relevant_sources
 end
 
```

Comment:
Will do. I think I just forgot the docstring rather than intentionally omitting it.